src/Controller/CRUDController.php

```diff
@@ -1197,11 +1197,12 @@ protected function redirectTo($object)
     }
 
     /**
+     * @final Since 4.0
      * Redirects the user to the list view.
      *
      * @return RedirectResponse
      */
-    final protected function redirectToList()
+    protected function redirectToList()
```

Comment:
What if the signature is incompatible? What if the behavior of our implementation differs from yours? I think we should not merge this, otherwise we will have to name new methods like this: `youSureAsHellDidNotPickThatNameRightRedirectToList`.